test/language/testdata/c_sharp/StepDefinitions.cs

```diff
@@ -2,11 +2,27 @@
 
 namespace StepDefinitions
 {
-    [Binding]
-    public class BowlingSteps
-    {
-        [Given(@"^a regexp$")]
-        public void a_regexp() {
-        }
-    }
+	[Binding]
+	public class BowlingSteps
+	{
+		[Given(@"^a regexp$")]
+		public void ARegexp() 
+		{
+		}
+
+		[When(@"I have Regex parameters like (\d+) and (\d+)")]
```

Comment:
Please remove this or make it consistent with step definitions for the other languages. Ref point 4 in [CONTRIBUTING.md](https://github.com/cucumber/language-service/blob/main/CONTRIBUTING.md#add-support-for-a-new-programming-language).

There is a test that expects exact expressions: https://github.com/cucumber/language-service/blob/b7e8b98352d76a10acbd9edc8ddc193fb7fedcc7/test/language/ExpressionBuilder.test.ts#L58

Reply:
I see. I'll update this. What are your thoughts on this `[When(@"I have Regex parameters like (\d+) and (\d+)")]` expression out of interest? It is a common pattern in the Specflow generated step definitions (using the Visual Studio Specflow extension), so I'm wondering if it's OK to not have it included here as part of the testing.